src/test/utils.ts

```diff
@@ -30,14 +30,16 @@ export const runQuery = (
     xForwardedFor: "123.456.789",
   }
 
-  return graphql(schema, query, rootValue, context).then(result => {
-    if (result.errors) {
-      const error = result.errors[0]
-      throw error.originalError || error
-    } else {
-      return result.data
-    }
-  })
+  return graphql(schema, query, rootValue, context)
+    .then(result => {
+      if (result.errors) {
+        const error = result.errors[0]
+        throw error.originalError || error
```

Comment:
without the `catch` down below this is the complete output of the error introduced above: 

<img width="356" alt="screen shot 2018-11-28 at 4 41 07 pm" src="https://user-images.githubusercontent.com/236943/49191704-77463300-f32c-11e8-8379-0376ef72190d.png">

(The stack trace provides no originating file location.)